collections/sample.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+import { randomNumber } from "./_utils.ts";
+
+/**
+ * Returns a random element from the given array.
+ *
+ * Example:
+ *
+ * ```ts
+ * import { sample } from "./sample.ts"
+ * import { assert } from "../testing/asserts.ts";
+ *
+ * const numbers = [1, 2, 3, 4];
+ * const random = sample(numbers);
+ *
+ * assert(numbers.includes(random));
```

Comment:
this seems causing type error in doc testing. you can check that with the command
`deno test --doc collections/sample.ts`